### PR TITLE
Update to pitivi 2020.09.0

### PR DESCRIPTION
**Pull request by thiblahute**

### CODE REVIEW

File: org.pitivi.Pitivi.json [hunk 1]
```diff
@@ -46,176 +46,70 @@
         }
     },
     "modules" : [
+        "python3-ipdb.json",
+        "python3-nose2.json",
+        "python3-pre-commit.json",
+        "python3-pylint.json",
```
Comment (aleb) [inline org.pitivi.Pitivi.json:52]: These 4 are not needed

**Review by aleb — CHANGES_REQUESTED** [review]

**Comment by A6GibKm** [comment]
The new runtime should come with lame 3.100, you might remove line 136
```patch
-        "shared-modules/lame/lame-3.99.5.json",
```

**Comment by A6GibKm** [comment]
@aleb  could you please consider giving support to the 3.38 GNOME runtime for a future 20.09.1 release?

File: org.pitivi.Pitivi.json [hunk 2]
```diff
@@ -370,30 +339,31 @@
...
             "config-opts" : [
-                "-Ddisable_gtkdoc=true"
+                "-Ddoc=disabled"
             ],
             "sources" : [
                 {
                     "type" : "git",
-                    "branch" : "1.14.2",
+                    "branch" : "1.18.0",
                     "url" : "git://anongit.freedesktop.org/gstreamer/gstreamer",
```
Comment (A6GibKm) [inline org.pitivi.Pitivi.json:348]: Isn't https://gitlab.freedesktop.org/gstreamer.git the correct url?

**Comment by thiblahute** [comment]
> @thiblahute @aleb could you please consider giving support to the 3.38 GNOME runtime for a future 20.09.1 release?

We might but we should have the change happen upstream and tested before we make the change in flathub.

**Comment by thiblahute** [comment]
> The new runtime should come with lame 3.100, you might remove line 136
> 
> ```diff
> -        "shared-modules/lame/lame-3.99.5.json",
> ```
> 
> Edit: This is the case for GNOME 3.38. I don't know about 3.36. In any case have a look at https://gitlab.gnome.org/GNOME/pitivi/-/blob/2020.09.0/build/flatpak/org.pitivi.Pitivi.json#L135.

I did check and it is in 3.36 too.

**Comment by hfiguiere** [comment]
When I launch this build, I get:

```
Failed to initialize modules
Traceback (most recent call last):
  File "/app/bin/pitivi", line 152, in <module>
    _initialize_modules()
  File "/app/bin/pitivi", line 129, in _initialize_modules
    initialize_modules()
  File "/app/lib/pitivi/python/pitivi/check.py", line 399, in initialize_modules
    if validate.init() and "--inspect-action-type" in sys.argv:
  File "/app/lib/pitivi/python/pitivi/utils/validate.py", line 558, in init
    Gst.info("Adding pitivi::wrong-window-creation")
  File "/usr/lib/python3.7/site-packages/gi/module.py", line 132, in __getattr__
    self.__name__, name))
AttributeError: 'gi.repository.Gst' object has no attribute 'info'
```

It fails.

**Comment by A6GibKm** [comment]
I will add
```
** (gst-plugin-scanner:5): CRITICAL **: 22:09:23.541: Couldn't g_module_open libpython. Reason: /usr/lib/x86_64-linux-gnu/libpython3.7m.so: cannot open shared object file: No such file or directory
```
To the issue above.

**Comment by thiblahute** [comment]
Thanks for testing, updated now, let's see how it goes :-)

**Comment by A6GibKm** [comment]
Now
```
ERROR - The following hard dependencies are unmet:
==================================================
- Peas not found on the system
```
It might be the case that it the dependency is missing in the meson file too? For convenience: https://gitlab.gnome.org/GNOME/pitivi/-/blob/master/build/flatpak/org.pitivi.Pitivi.json#L529.

**Comment by thiblahute** [comment]
Yeah, that is the reason, although it is interesting that libpeas is not in the GNOME Sdk.

**Comment by thiblahute** [comment]
Look all good to me, @aleb, do you want to give it a try and merge?

**Comment by thiblahute** [comment]
OK, getting it in.

**Pull request merged**